runtests

```diff
@@ -136,7 +139,7 @@ if __name__ == '__main__':
         import tests
         tests.tools.set_local_umake()
 
-    if not "all" in args.tests and len(args.tests) > 0:
+    if "all" not in args.tests and len(args.tests) > 0:
```

Comment:
Can you move all flake8/enhancement not related to this pull request in another PR? I prefer to have separated branches (I guess you can do a rebase and split that in 2)